redbot/core/dev_commands.py

```diff
@@ -336,17 +336,33 @@ async def mock(self, ctx, user: discord.Member, *, command):
 
     @commands.command(name="mockmsg")
     @checks.is_owner()
-    async def mock_msg(self, ctx, user: discord.Member, *, content: str):
+    async def mock_msg(self, ctx, user: Union[discord.Member, discord.User, int], *, content: str):
         """Dispatch a message event as if it were sent by a different user.
 
         Only reads the raw content of the message. Attachments, embeds etc. are
         ignored.
         """
+        if isinstance(user, int):
```

Comment:
What is the purpose of this? when will a user or member known to the user be just a int?